tests/importer/data/existing_eitems.json

```diff
@@ -157,9 +157,9 @@
   },
   {
     "pid": "eitemid-9",
-    "created_by": {"type": "user", "value":  "1"},
+    "created_by": {"type": "import", "value": "safari"},
     "document_pid": "docid-9",
-    "eitem_type": "AUDIOBOOK",
+    "eitem_type": "E-BOOK",
```

Comment:
AUDIOBOOK creation is already there, this was for updating existing AUDIOBOOK, I will add another item here to cover that if you think it might be necessary, but it will be similar to pre-existing update E-BOOK case, WDYT?

Reply:
I think it could be useful to check the exact case that was pointed out to us, if audiobook is created when there is already e-book of the same provider existing